src/services/offer-service.ts

```diff
@@ -15,13 +15,38 @@ import {
 import { baseService } from './base-service'
 
 export const OfferService = {
-  getOffers: async (params?: GetOffersParams): Promise<AxiosResponse> => {
-    const category = createUrlPath(URLs.categories.get, params?.categoryId)
-    const subject = createUrlPath(URLs.subjects.get, params?.subjectId)
-    return await axiosClient.get(`${category}${subject}${URLs.offers.get}`, {
-      params
+  getOffers: async (params?: GetOffersParams) => {
+    const categoryId = params?.categoryId
+    const subjectId = params?.subjectId
```

Comment:
You can try this approach, it will help to make code more readable

```suggestion
    const { categoryId, subjectId, ...restParams } = params
```

Reply:
I agree. Fixed.

Reply:
But sometimes categoryId and subject id will not be present in that params object

Reply:
Is it better to write like this in offer-service: const { categoryId, subjectId, ...restParams } = params || { } 
or simply pass empty object in tests?

Reply:
Fixed.